src/main/java/bio/terra/service/snapshot/flight/create/CreateSamGroupStep.java

```diff
@@ -0,0 +1,37 @@
+package bio.terra.service.snapshot.flight.create;
+
+import bio.terra.model.SnapshotRequestModel;
+import bio.terra.service.auth.iam.IamService;
+import bio.terra.service.snapshot.flight.SnapshotWorkingMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+
+public class CreateSamGroupStep implements Step {
```

Comment:
```suggestion
public class CreateFirecloudGroupStep implements Step {
```
I think we should name this to match what we're doing with duos group creation. (sam vs. firecloud)